accessibility-checker-engine/help/IBMA_Focus_MultiTab.mdx

```diff
@@ -8,11 +8,11 @@ import { CodeSnippet, Tag } from "carbon-components-react";
 <Row>
 <Column colLg={16} colMd={8} colSm={4} className="toolHead">
 
-### The component has more than one tabbable element
+### The component has more than one tabable element
```

Comment:
tabbable was correct

Reply:
agree.  These shoud not have been changes.  checking and correcting back